scripts/e2e-maestro.sh

```
@@ -111,7 +131,7 @@ if [ "$PLATFORM" == "ios" ]; then
   # Return to project root
   cd "$SCRIPT_DIR"
 else
-  cd $EXAMPLE_DIR/android
+  cd "$EXAMPLE_DIR/android"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)